defopt.py

```diff
@@ -218,14 +219,33 @@ def _populate_parser(func, parser, parsers, short):
             if param.kind == param.VAR_POSITIONAL:
                 kwargs['action'] = 'append'
                 kwargs['default'] = []
-        if inspect.isclass(type_.type) and issubclass(type_.type, Enum):
+        make_tuple = member_types = None
+        if _is_generic_type(type_.type, Tuple):
+            make_tuple = tuple
+            member_types = type_.type.__args__
+            kwargs['nargs'] = len(member_types)
+        elif (inspect.isclass(type_.type) and issubclass(type_.type, tuple)
+              and hasattr(type_.type, '_fields')
+              and hasattr(type_.type, '_field_types')):
```

Comment:
On second thought perhaps it doesn't belong there, it's not really like the others at all.